docs/src/installation.md

```diff
@@ -125,7 +125,7 @@ specified by using the `SecondOrderCone` set.
 
 ### Gurobi
 
-￼Requires a working installation of Gurobi with an activated license (free for
+Requires a working installation of Gurobi with an activated license (free for
```

Comment:
I fail to see what changed

Reply:
Check http://www.juliaopt.org/JuMP.jl/dev/installation/#MOSEK-1
There is a weird character

Reply:
I don't see it, where is it ?

Reply:
I see it in Firefox, not Chrome